README.md

```diff
@@ -93,16 +103,60 @@ A helper class `Drawing` exists to help with performing common drawing operation
 See the [`examples/surface.js`](./examples/surface.js) example for more information
 on how to use this API.
 
+
+
+## Limited support for the legacy Launchpad MK1
+
+[limited-support-mk1]: #limited-support-for-the-legacy-launchpad-mk1 "Jump to section"
+
+Launchpad MK1 offers less functionality over newer models and therefore has some limitations.
+The implementation for Launchpad MK1 aims to be forward compatible with functionality of this package that
+was added with newer models in mind, though there are some differences as specified in this section.
+
+
+### Colors
+
+Buttons on Launchpad MK1 only have two LEDs, red and green, with four states each:
+- `0` off.
+- `1` low brightness.
+- `2` medium brightness.
+- `3` full brightness.
+
+This means color values used by methods `lp.setButtonColor()`, `lp.flash()` and `lp.pulse()`
+  are restricted to the following values:
+- `[r, g]`, a tuple of red and green values in range `0..3`.
+- `[r, g, b]`, a tuple of red, green and blue values in range `0..3`. The value for blue is ignored.
+- `number` in range `0..63`, specifying a *Velocity* value as specified in the developer reference.
+  *Velocity* values can be calculated by using method `lp.velocity()`.
+
+
+### Flashing buttons
+
+Buttons on Launchpad MK1 can only **pulse** between off and a specified color, it does not support **flash**ing
+  between two specified colors. This results in method `lp.flash()` only accepting one color argument and
+  thus providing the exact same signature and functionality as `lp.pulse()`.
+
+
+
 ## TODO
 
 - Add support for the same launchpads as launchpad.py
 
-### Links
 
-- [Launchpad developer manual](https://resource.novationmusic.com/support/product-downloads?product=Launchpad)
+## Resources
+
+Cached in this repository:
+- [Launchpad (MK1) programmers reference](./resources/launchpad-mk1-programmers-reference.pdf)
+- [Launchpad MK2 programmers reference](./resources/launchpad-mk2-programmers-reference-v1.03.pdf)
+- [Launchpad Mini MK3 programmers reference](./resources/launchpad-mk3-mini-programmers-reference.pdf)
```

Comment:
My only concern is whether or not this is allowed license wise

Reply:
I understand, the reason behind it is to cache those documents in case they become unavailable as sadly is often the case with older niche products. Novation is currently doing a good job on keeping them available though.

I see those documents have been archived on the wayback machine too, so referencing them is probably a better solution.